source/howto/CreateVMFromCommandLine.rst

```diff
@@ -124,22 +89,19 @@ To see the list of networks available, run the command:-
 ######################################################
 Putting it all together to create a new Instance
 ######################################################
-
-Here is an example command, putting together information from the previous commands:-
+Please see the openstack documentation for the full set of parameters: https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/server.html#server-create
 
 .. code-block:: bash
 
-  openstack server create --flavor m1.tiny --image ScientificLinux-7-NoGui --nic net-id=Internal --security-group default --key-name xbe91637 test_2018-10-29_1511
-
-…where flavor and image are from the previous commands used, net_id is the name of the Network to be used (note you can use the actual Net_ID number instead if preferred – it can make things faster!). Security group is defining the specific security group, and key-name, chooses the ssh keypair to include when creating the host. “test_2018-10-29_1511” is the name of the host that is being created – known within openstack.
-Some useful extras
-Adding --timing after the openstack command provides some statistics of how quickly various calls are being completed. You will see the usual host creation data, but at the end, you will also see the response times of each openstack API module.
+  openstack server create
 
 .. code-block:: bash
 
-  openstack --timing server create --flavor m1.tiny --image Ubuntu-Xenial-NoGui --nic net-id=Internal --security-group default --key-name xbe91637 test_2018-10-30_1357
+Here is an example command, putting together information from the previous commands:-
 
+.. code-block:: bash
 
+  openstack server create --flavor <flavor ID/name> --image <image ID/name> --network <network> --security-group default --key-name <key name> <server name>
```

Comment:
I would still give a single example and just make it a bit more generic:

```openstack server create --flavor l3.nano --image Ubuntu-22.04 --network Internal --security-group default --key-name key_name the_servers_name```

We should also link to creating an SSH key for the key name part